pkgs/tools/filesystems/dbxfs/default.nix

```diff
@@ -0,0 +1,101 @@
+{ lib, fetchgit, python3Packages, fusePackages }:
+
+let
+  block-tracing = python3Packages.buildPythonPackage rec {
+    pname = "block_tracing";
+    version = "1.0.1";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "9faa009a702a8f2f605ebb78314d5ca2a2a93543d061038a3d3a978c93385e68";
+    };
+  };
+  privy = python3Packages.buildPythonPackage rec {
+    pname = "privy";
+    version = "6.0.0";
+    format = "wheel";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      format = "wheel";
+      sha256 = "e68679bb4006ce83206d9a7af1158cf4d56a2ed6861ee39276907be618dfb8d9";
+    };
+
+    propagatedBuildInputs = [
+      python3Packages.cryptography
+      python3Packages.argon2_cffi
+    ];
+  };
+  fusepyng = python3Packages.buildPythonPackage rec {
+    pname = "fusepyng";
+    version = "1.0.7";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "sha256-i09u+B6GAPI9p1CRaayyYVWC7xFtRqKhrUt+Uw2PiZ8=";
+    };
+
+  };
+  userspacefs = python3Packages.buildPythonPackage rec {
+    pname = "userspacefs";
+    version = "2.0.5";
+
+    src = python3Packages.fetchPypi {
+      inherit version;
+      inherit pname;
+      sha256 = "sha256-XW9f+m05SI8cdDfF6q6Pq/SRbKzqtIOjFzoY5nibGGw=";
+    };
+
+    buildInputs = [
+      fusePackages.fuse_2
+    ];
+
+    # crashes looking for libfuse in build_ext (?)
+    doCheck = false;
+
+    propagatedBuildInputs = [
+      fusepyng
+    ];
+  };
+in
+python3Packages.buildPythonApplication rec {
+  pname = "dbxfs";
+  version = "1.0.58";
+
+  src = fetchgit {
+    url = "https://thelig.ht/code/dbxfs";
+    deepClone = true; # this repo doesn't support shallow cloning
+    # rev = "refs/tags/v${version}"; # this doesn't seem to work, getting "Unrecognized git object type: tag"
+    rev = "97960fe59b75194d1b00388083968e7aaaf7a772";
+    sha256 = "sha256-GW1b9howk+/QDQf05L9ZbOduKyBTqJgw1pJHp8cY7vk=";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    python3Packages.keyring
+    python3Packages.keyrings-alt
+    python3Packages.dropbox
+    python3Packages.appdirs
+    python3Packages.sentry-sdk
+    block-tracing
+    privy
+    userspacefs
+  ];
+
+  buildInputs = [
+    fusePackages.fuse_2
+  ];
+
+  meta = with lib; {
+    description = "Allows you to mount your Dropbox folder as if it were a local filesystem or SMB share";
+    homepage = "https://thelig.ht/code/dbxfs/";
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
+    maintainers = [ maintainers.mausch ];
+  };
+}
+
```

Comment:
One final new line to much.

Reply:
Fixed in ca1cbcfe0a03bfb35d78d41d55cf597b8e1cb068